ci/cleanup

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# NOTE: The script should be executed from the repo directory
+
+# SCRIPT SUMMARY
+# --------------
+# 1. Deletes the minikube clusterrole
+# 2. Clears folders with caches and configurations
+
+# 0.
+source ci/0-setup-env
+
+# Set shell options
+# -e : Exit immediately if a command exits with a non-zero status.
+# -u : Treat unset variables as an error when substituting.
+# -x : Print commands and their arguments as they are executed.
+set -eux
+
+# 1.
+minikube delete
+
+# 2.
+rm -rf .kube/ .minikube/ .helm/
```

Comment:
we need to make sure we are absolutely in the right directory before we run this. Maybe test that `.kube` etc exist and that some other directories and files we know should be in the repo at a certain place are exactly where we think they are so this script can be sure it is not about to `rm -rf` something the user did want to keep :)

Reply:
In fact, I think there's no need to delete files in a cleanup script. We already have that with `git clean`, so we can leave our own cleanup as deleting the test cluster, since that's the only thing being created that wouldn't normally get cleaned up.